docs/dogfooding.md

```diff
@@ -144,4 +169,159 @@ spec:
+kubectl wait kafka/my-cluster --for=condition=Ready --timeout=300s -n kafka 
+```
+
+A [Knative Channel](https://github.com/knative-sandbox/eventing-kafka) is installed next:
+
+```shell
+# Install the Kafka "Consolidated" Channel
+kubectl apply -f https://storage.googleapis.com/knative-nightly/eventing-kafka/latest/channel-consolidated.yaml
+
+# Edit the "config-kafka" config-map in the "knative-eventing" namespace
+# Replace "REPLACE_WITH_CLUSTER_URL" with my-cluster-kafka-bootstrap.kafka:9092/
+kubectl edit cm/config-kafka -n knative-eventing
+```
+
+Install the [Knative Kafka Broken](https://knative.dev/docs/install/eventing/install-eventing-with-yaml/#optional-install-a-broker-layer)
+following the official guide:
+
+```shell
+# Kafka Controller
+kubectl apply -f https://github.com/knative-sandbox/eventing-kafka-broker/releases/download/knative-v1.0.0/eventing-kafka-controller.yaml
+
+# Kafka Broken Data plane
+kubectl apply -f https://github.com/knative-sandbox/eventing-kafka-broker/releases/download/knative-v1.0.0/eventing-kafka-broker.yaml
+```
+
+### Kafka UI
+
+The [Kafka UI](https://github.com/provectus/kafka-ui) allows viewing and searching for events stored by Kafka.
+Events are retained by Kafka for some time (but not indefinitely), which helps when debugging event based integrations.
+The Kafka UI allows managing channels and creating new events, so it is not publicly accessible. To access
+the Kafka UI, port-forward the service port:
+
+```shell
+# Set up port forwarding
+export POD_NAME=$(kubectl get pods --namespace kafka -l "app.kubernetes.io/name=kafka-ui,app.kubernetes.io/instance=kafka-ui" -o jsonpath="{.items[0].metadata.name}")
+kubectl --namespace kafka port-forward $POD_NAME 8080:8080
+
+# Point the browser to http://localhost:8080
+```
+
+The Kafka UI is installed via an helm chart as recommended in the [Kubernetes installation guide](https://github.com/provectus/kafka-ui#running-in-kubernetes).
+
+```shell
+helm install kafka-ui kafka-ui/kafka-ui --set envs.config.KAFKA_CLUSTERS_0_NAME=my-cluster --set envs.config.KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS=my-cluster-kafka-bootstrap:9092 --set envs.config.KAFKA_CLUSTERS_0_ZOOKEEPER=my-cluster-zookeeper-nodes:2181 --namespace kafka
+```
+
+### CloudEvents Producer
+
+Tekton Pipelines is the only `CloudEvents` producer in the cluster. It's [configured](../)tekton/cd/pipeline/overlays/dogfooding/config-defaults.yaml) to send all events to the broker:
+
+```yaml
+data:
+  default-cloud-events-sink: http://kafka-broker-ingress.knative-eventing.svc.cluster.local/default/default
+```
+
+### CloudEvents Consumers
+
+`CloudEvents` are consumed from the broker via a Knative Eventing CRD called `Trigger`.
+The `dogfooding` cluster is setup so that all `TaskRun` start, running and finish events are forwarded from the
+broker to the `tekton-events` event listener, in the `default` namespace.
+This initial filtering of events allows to reduce the load on the event listener.
```

Comment:
This is nice. Did we have performance issues due to heavy load on the EL?

Reply:
I don't have hard numbers on this, but my feeling is that the el pod was coping OK with the load.
But I'm happy to reduce the noise in the logs and simplify the CEL filtering.

Reply:
yeah there is definitely a lot of logs.